resources/schema.gql

```diff
@@ -1119,6 +1130,7 @@ input FlowRunConfiguration @oneOf {
 	schedule: ScheduleInput
 	batching: BatchingConditionInput
 	compaction: CompactionConditionInput
+	ingest: IngestConditionInput
```

Comment:
You replaced `schedule` in `FlowConfiguration` output enum with `ingest`, but kept both in this one. Is this intentional?

Reply:
Yeah, in GQL I removed it